index.html

```diff
@@ -89,6 +96,9 @@ <h4>Instructions:</h4>
             <option value="surface">surface</option>
           </select>
           <button id="protein_plot_download" type="button" class="btn btn-light float-right"><img src="/assets/images/download.svg" /> Download PNG</button>
```

Comment:
I wish the "Download PNG" button was positioned to the right of the protein panel like the download buttons in the other panels.

Reply:
Yes! Consistency is good. Changed!